### ISSUE TITLE
DatabaseError: (DatabaseError) ORA-12704: character set mismatch for Oracle gers = false

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1073]
_Imported from trac ticket 1073.
Created by **kbower**
Opened in trac: Sep 20, 2011, 10:23:59 AM
Last modified in trac: Jan 3, 2012, 6:16:17 PM_
***
Exception immediately from flex after log in.

**Comment by kbower** [comment]
_**kbower** trac comment on Jan 3, 2012, 5:28:01 PM:_

Occurs at the 'Z'
```
WHERE rp__a.productid = products.productid AND rp__a.siteid = :siteid_1 AND rp__a.effectivedate <= :effectivedate_1 AND rp__a.endingdate >= :endingdate_1), 'Z'), coalesce((SELECT min('A' || to_char(promoprice__a.price, :to_char_2) || ',' || length(promo__a.description) || ',' || promo__a.description || promo__a.promotionid) AS min_2
                                                                                                                                                            *
ERROR at line 11:
ORA-12704: character set mismatch
```

**Comment by kbower** [comment]
_**kbower** trac comment on Jan 3, 2012, 6:16:17 PM:_

(Changeset [[2104]](../commit/1002d3cc61420533101b309582a51255b91e02b5))
 * fix ORA-12704 for gers = false (fixes #1073)

**Comment by kbower** [comment]
_**kbower** trac status change on Jan 3, 2012, 6:16:17 PM:_ **closed**
